File: domain/src/main/java/no/nav/melosys/domain/dokument/soeknad/ArbeidsgiversBekreftelse.java
package no.nav.melosys.domain.dokument.soeknad;

import java.time.LocalDate;
import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;

import no.nav.melosys.domain.dokument.jaxb.LocalDateXmlAdapter;

public class ArbeidsgiversBekreftelse {

    public boolean arbeidsgiverBekrefterUtsendelse;
    public boolean arbeidstakerAnsattUnderUtsendelsen;
    public boolean erstatterArbeidstakerenUtsendte;
    public boolean arbeidstakerTidligereUtsendt24Mnd;
    public boolean arbeidsgiverBetalerArbeidsgiveravgift;
    public boolean trygdeavgiftTrukketGjennomSkatt;
    @XmlJavaTypeAdapter(LocalDateXmlAdapter.class)
    public LocalDate trygdeavgiftTrukketGjennomSkattDato;
}

File: domain/src/main/kotlin/no/nav/melosys/domain/mottatteopplysninger/data/ArbeidsgiversBekreftelse.kt
package no.nav.melosys.domain.mottatteopplysninger.data

import java.time.LocalDate


class ArbeidsgiversBekreftelse {
    var arbeidsgiverBekrefterUtsendelse: Boolean? = null
    var arbeidstakerAnsattUnderUtsendelsen: Boolean? = null
    var erstatterArbeidstakerenUtsendte: Boolean? = null
    var arbeidstakerTidligereUtsendt24Mnd: Boolean? = null
    var arbeidsgiverBetalerArbeidsgiveravgift: Boolean? = null
    var trygdeavgiftTrukketGjennomSkatt: Boolean? = null
    var trygdeavgiftTrukketGjennomSkattDato: LocalDate? = null
}

